<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormJenis</class>
 <widget class="QWidget" name="FormJenis">
  <property name="geometry"><rect><x>0</x><y>0</y><width>860</width><height>580</height></rect></property>
  <property name="windowTitle"><string>Form Jenis</string></property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry"><rect><x>10</x><y>10</y><width>521</width><height>175</height></rect></property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0"><widget class="QLabel" name="lbl_kode"><property name="text"><string>kode</string></property></widget></item>
    <item row="0" column="1"><widget class="QLineEdit" name="edit_kode"/></item>
    <item row="1" column="0"><widget class="QLabel" name="lbl_nama"><property name="text"><string>nama</string></property></widget></item>
    <item row="1" column="1"><widget class="QLineEdit" name="edit_nama"/></item>
    <item row="2" column="0"><widget class="QLabel" name="lbl_keterangan"><property name="text"><string>keterangan</string></property></widget></item>
    <item row="2" column="1"><widget class="QLineEdit" name="edit_keterangan"/></item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnSimpan"><property name="geometry"><rect><x>560</x><y>20</y><width>121</width><height>28</height></rect></property><property name="text"><string>Simpan</string></property></widget>
  <widget class="QPushButton" name="btnUbah"><property name="geometry"><rect><x>560</x><y>55</y><width>121</width><height>28</height></rect></property><property name="text"><string>Ubah</string></property></widget>
  <widget class="QPushButton" name="btnHapus"><property name="geometry"><rect><x>560</x><y>90</y><width>121</width><height>28</height></rect></property><property name="text"><string>Hapus</string></property></widget>
  <widget class="QPushButton" name="btnBersih"><property name="geometry"><rect><x>560</x><y>125</y><width>121</width><height>28</height></rect></property><property name="text"><string>Bersih</string></property></widget>
  <widget class="QLineEdit" name="editCari"><property name="geometry"><rect><x>10</x><y>200</y><width>311</width><height>28</height></rect></property><property name="placeholderText"><string>Cari...</string></property></widget>
  <widget class="QTableWidget" name="table">
   <property name="geometry"><rect><x>10</x><y>235</y><width>831</width><height>331</height></rect></property>
   <property name="columnCount"><number>3</number></property>
   <property name="rowCount"><number>0</number></property>
   <column><property name="text"><string>kode</string></property></column>
   <column><property name="text"><string>nama</string></property></column>
   <column><property name="text"><string>keterangan</string></property></column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
